src/Components/Organisms/ReservationCard/ReservationCard.tsx

```diff
@@ -1,8 +1,8 @@
 import { Box, Button, Card, Chip, Typography } from '@mui/material';
 import { useTranslation } from 'react-i18next';
 
+import { GenerateDateProps } from '../../../Tools/reservationTools';
```

Comment:
I think its a function not a component, maybe small letter was ok though ?

Reply:
I can't use hook useTranslation in it earlier :) So it's the only reason... :)